launcher/src/main/java/org/wildfly/core/launcher/StandaloneCommandBuilder.java

```diff
@@ -126,7 +126,13 @@ public StandaloneCommandBuilder addJavaOption(final String jvmArg) {
                     }
                     break;
                 case SECURITY_MANAGER_PROP:
-                    setUseSecurityManager(true);
+                    if (ALLOW_VALUE.equals(argument.getValue())) {
+                        // [WFCORE-5778] java.security.manager system property with value "allow" detected.
+                        // It doesn't mean SM is going to be installed but it indicates SM can be installed dynamically.
+                        setUseSecurityManager(false);
+                    } else {
+                        setUseSecurityManager(true);
+                    }
```

Comment:
Just a minor comment but this could be simplified to `setUseSecurityManager(!ALLOW_VALUE.equals(argument.getValue()));` like in the `DomainCommandBuilder`.

Reply:
Fixed.